UnityClient/Assets/Scripts/TowerDefence/WaveSpawner.cs

```diff
@@ -8,50 +8,76 @@
 
 public class WaveSpawner : MonoBehaviour
 {
-    public Transform enemyPrefab;
+    public static int EnemiesAlive = 0;
```

Comment:
`static` is strictly prohibited for use cases like this. It is brittle and not good design. Please redesign this to not be `static`. Thanks. Assuming this is public-facing data you might also want to consider this to be a property, or whatever fits your new design.

Reply:
tried turning the static into get and set but did not work